### PR TITLE
[DOCS] technical term create expectation configuring profilers

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Updates documentation under the "Create Expectations: How-to guides: Configuring Profilers" section of the ToC.
    - Brings docs in line with documentation and how-to guide standards.
    - Adds technical term tags to docs.

After submitting your PR, CI checks will run and @ge-cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [X] I have made corresponding changes to the documentation

Thank you for submitting!

### CODE REVIEW

File: docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md [hunk 1]
```diff
@@ -14,25 +15,30 @@ While the contents of this document accurately reflect the state of the feature,
...
 - Move the unzipped directory of `csv` files into the `data` directory that you created in Step 1
 
-### 3. Setting up your Datasource
-- Follow the steps in the [How to connect to data on a filesystem using Pandas](../../../guides/connecting_to_your_data/filesystem/pandas.md). For the purpose of this tutorial, we will work from a `yaml` to set up your datasource config. When you open up your notebook to create and test and save your datasource config, replace the config docstring with the following docstring:
+### 3. Set up your Datasource
+
+- Follow the steps in the [How to connect to data on a filesystem using Pandas](../../../guides/connecting_to_your_data/filesystem/pandas.md). For the purpose of this tutorial, we will work from a `yaml` to set up your <TechnicalTag tag="datasource" text="Datasource" /> config. When you open up your notebook to create and test and save your Datasource config, replace the config docstring with the following docstring:
+-
```
Comment (austiezr) [inline docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md]: Dangling bullet

File: docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md [hunk 2]
```diff
@@ -77,72 +87,89 @@ Note that at the time of writing this document, `1.0` is the only supported conf
...
 
-Finally, you would use an ExpectationConfigurationBuilder to actually build your `expect_table_row_count_to_be_between` expectation, where the Domain is the Domain returned by your TableDomainBuilder (your entire table), and the `min_value` and `max_value` are Parameters returned by your NumericMetricRangeMultiBatchParameterBuilder.
 ```yaml file=../../../../tests/integration/docusaurus/expectations/advanced/multi_batch_rule_based_profiler_example.py#L36-L44
 ```
 You can see here that we use a special `$` syntax to reference `variables` and `parameters` that have been previously defined in our config. You can see a more thorough description of this syntax in the  docstring for [`ParameterContainer` here](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/rule_based_profiler/types/parameter_container.py).
 
 - When we put it all together, here is what our config with our single `row_count_rule` looks like:
+-
```
Comment (austiezr) [inline docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md]: Dangling bullet

**Review by austiezr — APPROVED** [review]
couple dangling bullets; otherwise LGTM! ✅

File: docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md (hunk 1 above)
Comment (Rachel-Reverie) [reply docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md]: Fixed.

File: docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md (hunk 2 above)
Comment (Rachel-Reverie) [reply docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md]: Also fixed.  Thanks for the catches!

**Pull request merged**